File: libs/common/models/vw-session-of-study.ts



export class VwSessionOfStudy {
    public sessionOfStudyId: number = 0;
    public sessionOfStudyGuid: string;
    public fundingYearCode: string = '';
    public fundingYear: string = '';
    public displayOnSfsflag: boolean;
    public studyYear: number;
    public sessionStartDate: Date;
    public sessionEndDate: Date;
    public specializationId: number;

    constructor(init? : Partial<VwSessionOfStudy>) {
        
        Object.assign(this, init);
    }
}

File: apps/institution-api/Models/EntityFramework/VwSessionOfStudy.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace ProviderApi.Models.EntityFramework
{
    public partial class VwSessionOfStudy
    {
        public int SessionOfStudyId { get; set; }
        public Guid? SessionOfStudyGuid { get; set; }
        public string FundingYearCode { get; set; }
        public string FundingYear { get; set; }
        public bool? DisplayOnSfsflag { get; set; }
        public decimal? StudyYear { get; set; }
        public DateTime? SessionStartDate { get; set; }
        public DateTime? SessionEndDate { get; set; }
        public int? SpecializationId { get; set; }
    }
}
